addons/armaos/functions/fnc_os_echo.sqf

```diff
@@ -34,7 +34,24 @@ private _text = _ae3OptsThings joinString " ";
 
 if (_backslashInterpretion) then
 {
-    _text = _text splitString "\n";
```

Comment:
See https://community.bistudio.com/wiki/BIS_fnc_splitString using the `byWord` option

Reply:
Thanks for the suggestion. I've searched for something like that but obviosly you had more luck. I changed this.